sdk/react/style/controlBarStyles.json

```diff
@@ -32,7 +32,7 @@
     "fontFamily": "AvenirNext-DemiBold",
   },
   "completeTimeStyle": {
-    "flex":0,
+    "flex":1,
```

Comment:
reverted back. and added the upnext image stylings too the same branch. Please review.